### PR TITLE
Feature/ui2

### PR DESCRIPTION
**Pull request by EKCodeDevAcc**

### CODE REVIEW

File: webserver/pkgpkr/static/main.js [hunk 1]
```diff
@@ -15,18 +15,39 @@ document.addEventListener('DOMContentLoaded', () => {
...
+        $('#scoreModal').modal('hide');
+    })
 });
 
+function myFunction() {
+  var element = document.getElementsByClassName("pageloader");
+  element[0].classList.add("hidden");
+}
+
+// window.addEventListener("load", function() {
```
Comment (djsauble) [inline webserver/pkgpkr/static/main.js:38]: Delete this commented code

File: webserver/pkgpkr/webservice/templates/webservice/recommendations.html [hunk 2]
```diff
@@ -7,102 +7,109 @@
...
                         </td>
                     </tr>
                 </table>
                 <div class="recommendation-filter">
                     <p class="control has-icons-right">
                         <input id="recommendationFilter" class="input" type="text" placeholder="Search">
                         <span class="icon is-small is-right">
-            <i class="fas fa-search"></i>
-          </span>
+                            <i class="fas fa-search"></i>
```
Comment (djsauble) [inline webserver/pkgpkr/webservice/templates/webservice/recommendations.html:64]: Indent is wrong on these two lines.

File: webserver/pkgpkr/webservice/templates/webservice/recommendations.html [hunk 3]
```diff
@@ -113,15 +120,15 @@ <h4 class="modal-title" id="myModalLabel">Modal title</h4>
...
+                            return '<a target="package_details" href="' + row[2] + '">' + data + '</a>'
                         },
                         "width": "10em"
                     },
                     // PkgPkr Score
                     {
                         "targets": 3,
                         "render": function (data, type, row, meta) {
-                            return '<img class="pkgpkrscore_img" src="/static/images/' + data + '.png"><p class="pkgpkrscore_value">' + data + '</p>'
+                            return '<img class="pkgpkrscore_img" src="/static/images/' + data + '.png" style="width:100px;"><p class="pkgpkrscore_value">' + data + '</p>'
```
Comment (djsauble) [inline webserver/pkgpkr/webservice/templates/webservice/recommendations.html:131]: This style should go in styles.css

File: webserver/pkgpkr/webservice/views.py [hunk 4]
```diff
@@ -129,6 +133,13 @@ def repositories(request):
         'one_or_more_with_dependencies': one_or_more_with_dependencies
     })
 
+def manual_input(request):
+
+
+
+    return render(request, "webservice/manual-input.html", {})
+
+
```
Comment (vpopil) [inline webserver/pkgpkr/webservice/views.py:142]:
```suggestion
```

**Pull request merged**